nixos/modules/services/search/elasticsearch-curator.nix

```diff
@@ -32,6 +32,7 @@ let
     curatorAction = pkgs.writeTextFile {
       name = "action.yaml";
       text = cfg.actionYAML;
+      types = types.lines;
```

Comment:
That should go on or about line 58, and it's singular `type` on the left hand side.